flows/wikibase_to_vespa.py

```diff
@@ -0,0 +1,187 @@
+import asyncio
+from datetime import datetime, timezone
+from pathlib import Path
+
+import polars as pl
+from prefect import flow
+
+from flows.utils import get_logger
+from knowledge_graph.cloud import AwsEnv, get_aws_ssm_param
+from knowledge_graph.concept import Concept
+from knowledge_graph.wikibase import WikibaseSession
+
+# SSM
+WIKIBASE_PASSWORD_SSM_NAME = "/Wikibase/Cloud/ServiceAccount/Password"
+WIKIBASE_USERNAME_SSM_NAME = "/Wikibase/Cloud/ServiceAccount/Username"
+WIKIBASE_URL_SSM_NAME = "/Wikibase/Cloud/URL"
+
+
+def concepts_to_dataframe(concepts: list[Concept]) -> pl.DataFrame:
+    """
+    Convert a list of Concepts to a Polars DataFrame for data lake storage.
```

Comment:
Is this meant to be:

```suggestion
    Convert a list of Concepts to a Polars DataFrame for Delta Lake storage.
```

Reply:
I ditched Delta Lake since it was a bit more complex, and am just using some simple timestamp partitions via DataFrame via Parquet via Polars!

Reply:
This was confusing since I had an update elsewhere to make, to remove the mention of Delta Lake.